napari/_qt/dialogs/qt_package_installer.py

```diff
@@ -130,28 +129,13 @@ def arguments(self) -> Tuple[str, ...]:
             args.append('-vvv')
         if self.prefix is not None:
             args.extend(['--prefix', str(self.prefix)])
-        elif running_as_bundled_app(
-            check_conda=False
-        ) and sys.platform.startswith('linux'):
-            args += [
-                '--no-warn-script-location',
-                '--prefix',
-                user_plugin_dir(),
-            ]
         return (*args, *self.pkgs)
 
     def environment(
         self, env: QProcessEnvironment = None
     ) -> QProcessEnvironment:
         if env is None:
             env = QProcessEnvironment.systemEnvironment()
-        combined_paths = os.pathsep.join(
-            [
-                user_site_packages(),
```

Comment:
This was only used in Briefcase, AFAIK. There's no `plugins/` installation directory anywhere else, right?